apps/web/src/app/globals.css

```
@@ -1,120 +1,375 @@
+  .ledger-body .draw-b::after,
+  .ledger-body .draw-b-hair::after {
+    background: #000;
+  }
+  .ledger-body a {
+    color: #000 !important;
+    text-decoration: none;
+  }
+  /* Keep entries from splitting awkwardly across page breaks. */
+  .cv-block {
+    break-inside: avoid;
+    page-break-inside: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)